source/distributed-omit.d.ts

```diff
@@ -0,0 +1,80 @@
+import type {KeysOfUnion} from './keys-of-union';
+
+/**
+Omits keys from a type, distributing the operation over a union.
+
+Typescript's `Omit` doesn't distribute over unions, which causes the following situation:
+
+```ts
+type A = {
+  discriminant: "A";
```

Comment:
Follow the existing code style. Tab-indentation and single-quotes.

Reply:
Done!